pyproject.toml

```diff
@@ -41,6 +41,8 @@ dependencies = [
     "nbclassic",
     # pyvo pin can be removed once astroquery's pyvo dep pulls in 1.5.3
     "pyvo>=1.5.3",
+    "s3fs>=2024.10.0",
+    "boto3"
```

Comment:
Open question for all: should we add an optional deps group for cloud work, or should this be lumped with Roman?

Reply:
Is Roman always going to be cloud? Or maybe this should be extra "cloud" spec instead of always required? But then the `Observations.enable_cloud_dataset()` cannot always be on if this is only optional.

Reply:
I believe most of the higher level end user products will be cloud-only.  Some of the Roman catalog products will be on prem.  We may also store some of the lower level calibration products on prem.